WKWebView.ios.js

```diff
@@ -254,6 +254,7 @@ var WKWebView = React.createClass({
     });
 
     var source = this.props.source || {};
+    source = Object.assign(source, { sendCookies: this.props.sendCookies });
```

Comment:
I believe this will copy by value. There isn't a runtime exception being thrown, but I will update it to `var source = Object.assign({}, this.props.source || {} , { sendCookies: this.props.sendCookies });`